frontend/src/utilities/NavData.tsx

```diff
@@ -97,31 +97,41 @@ const useHomeNav = (): NavDataItem[] =>
 
 const useDSProjectsNav = (): NavDataItem[] =>
   useAreaCheck(SupportedArea.DS_PROJECTS_VIEW, [
-    { id: 'dsg', label: 'Data Science Projects', href: '/projects' },
+    { id: 'dsg', label: 'Data science projects', href: '/projects' },
   ]);
 
+const useModelsNav = (): NavDataItem[] => [
+  {
+    id: 'models',
```

Comment:
How can I completely disable seeing the Models navbar entry? I have every models-related set to `disableModelServing: true, disableModelCatalog: true, disableFineTuning: true, disableModelRegistry: true, disableModelServing: true`, yet I still see the Models entry. DataScienceCluster has no model serving, registry etc components enabled either.